source/MaterialXCore/Document.cpp

```diff
@@ -1016,7 +1016,7 @@ void Document::upgradeVersion(bool applyFutureUpdates)
             if (param)
             {
                 nodedef->removeParameter(AXIS);
-                nodedef->addInput(AXIS, "vector3");
+                nodedef->addInput(AXIS, "vector3", "vector3");
```

Comment:
There are a few changes like this that should be using a boolean third argument and not a string. Think this was a mistake - be sure to fix all the locations.

Reply:
Thanks for catching this. Fixed and added in JS, Py, C++ tests.